app/views/users/piv_cac_authentication_setup/error.html.erb

```diff
@@ -8,10 +8,8 @@
   <%= @presenter.description %>
 </p>
 
-<%= render PageFooterComponent.new do %>
-  <% if MfaPolicy.new(current_user).two_factor_enabled? %>
-    <%= link_to t('links.cancel'), account_path %>
-  <% else %>
-    <%= link_to t('two_factor_authentication.choose_another_option'), authentication_methods_setup_path %>
-  <% end %>
-<% end %>
+<%= render ButtonComponent.new(
+      url: setup_piv_cac_url,
+      class: 'usa-button margin-top-2',
```

Comment:
Same note here about using the correct button size (big & wide). Also, `ButtonComponent` handles adding the `usa-button` class, so we don't need to add it.

```suggestion
<%= render ButtonComponent.new(
      url: setup_piv_cac_url,
      big: true,
      wide: true,
      class: 'margin-top-2',
```

Reply:
Agree with with removing `usa-button`, but based on the Figma, it looks like the width should be the width of the content.

| Suggested change | Figma |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/eacd217a-cc41-4c80-b2f7-9cbad1320946) | <img width="442" alt="image" src="https://github.com/user-attachments/assets/a604b61c-f53e-4fc3-8236-f33541403999"> |

Reply:
I think that might be a mistake in the Figma designs, since these page layouts typically use the "wide" button variant. Could you get some clarification from UX people on the team?

Reply:
Fixed in [e517e40](https://github.com/18F/identity-idp/pull/10969/commits/e517e403a84fd00a55920e7deb7069d44e88ddbd)